dependencies/O2Dependencies.cmake

```diff
@@ -105,6 +105,9 @@ set_package_properties(RapidJSON PROPERTIES TYPE REQUIRED)
 find_package(CURL)
 set_package_properties(CURL PROPERTIES TYPE REQUIRED)
 
+find_package(ONNXRuntime::ONNXRuntime CONFIG)
```

Comment:
I guess you want to protect it with BUILD_ANALYSIS?

Reply:
added